### ISSUE TITLE
RBQRealmNotificationManager not included in project

### ISSUE THREAD

**Issue by hsavit1** [Roobiq/RBQFetchedResultsController#59]
I can't run the demo project. there are no RBQRealmNotificationManager files

**Comment by bigfish24** [comment]
Sorry docs need to add the following commands to pull in the submodules after cloning:

`git submodule init`
`git submodule update`

**Comment by hsavit1** [comment]
ok thanks @bigfish24 . I got the project to run. Are there any extra bits of information that I should know about as far as installation is concerned in my own project? I'm trying to use the Swift Project

**Comment by hsavit1** [comment]
it looks like the SafeObject.swift class was not in the Cocoapod project

**Comment by bigfish24** [comment]
Ah that's a good point, the Cocoapods install needs updating to use the SafeObject pod. Oops!

**Comment by bigfish24** [comment]
Sorry, I take that back, it is including it, forgot Cocoapods handles submodules properly.

How did you do the install?

**Comment by bigfish24** [comment]
I just tested the example project using the normal Cocoapods install. I removed the linked files from the example app Xcode project directly and instead added `SwiftFetchedResultsController` to the pod file. I also removed the bridging header and added `import SwiftFetchedResultsController` to `MainTableViewController.swift`.

These changes make the example app use the library from Cocoapods versus linking to the files directly. Everything works and I see `SafeObject.swift` in the `SwiftFetchedResultsController` pod folder
